### PR TITLE
Upgrade videojs to 5.0.0

### LINKED ISSUE #10672: Videos throw adsloader error and don't autoplay
The videos linked from http://www.theguardian.com/lexus-create-amazing are set-up to auto-play in the mark-up, as per other non-commercial video content. When these pages load, however, they log an error related to the video player expecting an ad playlist which seems to prevent the auto-playing of the video.

The error is: AdsLoader error: AdError 1010: The ad playlist response was malformed or empty. which refers to [google.ima.AdError](https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdError.Type)

Testing locally, I can add the following in the [onAdsLoaderError_ listener](https://github.com/guardian/frontend/blob/157d7230cd1dbc1837ddb8d3c2b55de50862234d/static/src/javascripts/components/videojs-ima/videojs.ima.js#L219)

```
adContainerDiv.style.display = 'none';
      player.play();
```

as per [onAddError_ listener](https://github.com/guardian/frontend/blob/157d7230cd1dbc1837ddb8d3c2b55de50862234d/static/src/javascripts/components/videojs-ima/videojs.ima.js#L231) which seems to be similar to what [newer version of the script from Google does](https://github.com/googleads/videojs-ima/blob/master/src/videojs.ima.js#L258).

This masks the error, but obviously doesn't fix it - @sndrs @desbo @OliverJAsh - What are your thoughts on adding this in so that the autoplaying of ads throwing the AdsLoader still load as expected, while I investigate the root issue? Do you have any ideas on the root issue..?

This can also be seen on [this video](http://www.theguardian.com/science/video/2015/sep/28/super-blood-moon-red-lunar-eclipse-around-world-video)

### PR DESCRIPTION
**Pull request by rich-nguyen**
Upgrading videojs and all it's plugins to version 5.0.0. This fixes #10672.

The majority of the changes are caused by api changes within videojs, a transition to UMD format, and DOM variations in the enhanced video DOM tree.

@Calanthe

### CODE REVIEW

**Comment by jbreckmckye** [comment]
So, is this a change to the third party plugin itself? Are we going to make a PR to their project, or maintain our own fork?

**Comment by rich-nguyen** [comment]
the videojs system has not been forked, but many of the plugins have. We try to merge our forks where possible, but it depends on what kind of fix was needed. For many of these, it was the UMD header which was required.

File: static/src/javascripts/components/videojs-ima/videojs.ima.js [hunk 1]
```diff
@@ -99,6 +115,14 @@
...
           player.ima.onAdMuteClick_,
           false);
+      sliderDiv = document.createElement('div');
+      sliderDiv.id = 'ima-slider-div';
+      sliderDiv.addEventListener(
+          'mousedown',
+          player.ima.onAdVolumeSliderMouseDown_,
+          false);
+      sliderLevelDiv = document.createElement('div');
+      sliderLevelDiv.id = 'ima-slider-level-div';
```
Comment (Calanthe) [inline static/src/javascripts/components/videojs-ima/videojs.ima.js]: Why did you have to add his? Was it part of the Darren's work?

**Comment by ScottPainterGNM** [comment]
https://docs.google.com/document/d/1SIPaug4tXcv3hrksE9NVhxLv5fIP-b-kETh23C-PTfI/edit

File: grunt-configs/requirejs.js [hunk 2]
```diff
@@ -212,29 +212,13 @@ module.exports = function(grunt, options) {
                 out: options.staticTargetDir + 'javascripts/bootstraps/ophan.js'
             }
         },
-        video : {
+        media: {
```
Comment (OliverJAsh) [inline grunt-configs/requirejs.js]: And same for jspm world?

Comment (rich-nguyen) [inline grunt-configs/requirejs.js]: Good spot, fixed it. It's easy to forget bundled build in dev mode.

**Comment by rich-nguyen** [comment]
Thanks @ScottPainterGNM, went through that doc. Fixed the styling, the jspm build, and checked browsers on windows (they seemed fine after the jspm fix, and allowing flash). 

The 'no longer supported browser' message is an old decision we made to limit the video support matrix; it was roughly modelled on youtube's. And the video player's skinning 'pop' is an unpleasant side-effect of progessively enhancing a native element. So, things left to do:

- [ ] ipad3 still dodgy, check embed link.
- [ ] vpaid player is needed

**Comment by gtrufitt** [comment]
:+1: Code style sanity check :D

**Pull request merged**